### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by khamlyuk**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -10,19 +9,34 @@
...
 
-The export buttons are located within the [ExportButtons](./CS/DxDataGridExportingWithReports/Components/ExportButtons.razor) components. Each of them contains an [URI to this project](./CS/DxDataGridExportingWithReports/Pages/Index.razor#L32). The request with this URI will be processed by the mentioned middleware. Also, this URI contains DataGrid's options, so the created report will only contain data that is displayed in the grid after sorting and filtering are applied.
+The [ExportButtons](./CS/DxDataGridExportingWithReports/Components/ExportButtons.razor) component contains export buttons. Each of the export buttons contains an [URI to this project](./CS/DxDataGridExportingWithReports/Pages/Index.razor#L32). The URI contains the Data Grid options. That is why the created report contains only data that is visible in the grid after sorting and filtering operations. The **ExportMiddleware** processes the request with the URI.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
The [ExportButtons](./CS/DxDataGridExportingWithReports/Components/ExportButtons.razor) component contains export buttons. Each export button contains an [URI to this project](./CS/DxDataGridExportingWithReports/Pages/Index.razor#L32), and the URI contains the Data Grid options. The created report contains only data that is visible in the grid after sort and filter operations. The **ExportMiddleware** processes the request with the URI.

```

File: Readme.md [hunk 2]
```diff
@@ -10,19 +9,34 @@
...
+Use the [ReportHelper.CreateReport](./CS/DxDataGridExportingWithReports/Helpers/ReportHelper.cs#L9) method together with the [ExportToPdf(String)](https://docs.devexpress.com/XtraReports/DevExpress.XtraReports.UI.XtraReport.ExportToPdf(System.String-DevExpress.XtraPrinting.PdfExportOptions))/[ExportToXlsx(Stream)](https://docs.devexpress.com/XtraReports/DevExpress.XtraReports.UI.XtraReport.ExportToXls(System.IO.Stream-DevExpress.XtraPrinting.XlsExportOptions))/[ExportToDocx(Stream)](https://docs.devexpress.com/XtraReports/DevExpress.XtraReports.UI.XtraReport.ExportToDocx(System.IO.Stream-DevExpress.XtraPrinting.DocxExportOptions)) methods to create a report that is exported to the file of the corresponding type.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
Use the [ReportHelper.CreateReport](./CS/DxDataGridExportingWithReports/Helpers/ReportHelper.cs#L9) method with the [ExportToPdf(String)](https://docs.devexpress.com/XtraReports/DevExpress.XtraReports.UI.XtraReport.ExportToPdf(System.String-DevExpress.XtraPrinting.PdfExportOptions))/[ExportToXlsx(Stream)](https://docs.devexpress.com/XtraReports/DevExpress.XtraReports.UI.XtraReport.ExportToXls(System.IO.Stream-DevExpress.XtraPrinting.XlsExportOptions))/[ExportToDocx(Stream)](https://docs.devexpress.com/XtraReports/DevExpress.XtraReports.UI.XtraReport.ExportToDocx(System.IO.Stream-DevExpress.XtraPrinting.DocxExportOptions)) methods to create a report that is exported to the file of the corresponding type.

```

Comment (dirk-pieterse) [reply Readme.md]: Which corresponding type?

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 2 above)
Comment (khamlyuk) [reply Readme.md]: PDF, xlsx, or docx depending on the chosen method.

**Pull request merged**